sdk/storage/storage-common/review/storage-common.api.md

```diff
@@ -4,20 +4,155 @@
 
 ```ts
 
+import type { CompatResponse } from '@azure/core-http-compat';
 import type { HttpClient } from '@azure/core-rest-pipeline';
+import type { HttpPipelineLogLevel } from '@azure/core-http-compat';
+import type { PipelinePolicy } from '@azure/core-rest-pipeline';
+import type { RequestPolicy } from '@azure/core-http-compat';
+import type { RequestPolicyFactory } from '@azure/core-http-compat';
+import type { RequestPolicyOptionsLike } from '@azure/core-http-compat';
+import type { RestError } from '@azure/core-rest-pipeline';
+import type { WebResourceLike } from '@azure/core-http-compat';
+
+// @public
+export class AnonymousCredential extends Credential_2 {
+    create(nextPolicy: RequestPolicy, options: RequestPolicyOptionsLike): AnonymousCredentialPolicy;
+}
+
+// @public
+export class AnonymousCredentialPolicy extends CredentialPolicy {
+    constructor(nextPolicy: RequestPolicy, options: RequestPolicyOptionsLike);
+}
+
+// @public
+export abstract class BaseRequestPolicy implements RequestPolicy {
+    protected constructor(
+    _nextPolicy: RequestPolicy,
+    _options: RequestPolicyOptionsLike);
+    log(logLevel: HttpPipelineLogLevel, message: string): void;
+    readonly _nextPolicy: RequestPolicy;
+    readonly _options: RequestPolicyOptionsLike;
+    abstract sendRequest(webResource: WebResourceLike): Promise<CompatResponse>;
+    shouldLog(logLevel: HttpPipelineLogLevel): boolean;
+}
 
 // @public
 export class BufferScheduler {
     constructor(readable: NodeJS.ReadableStream, bufferSize: number, maxBuffers: number, outgoingHandler: OutgoingHandler, concurrency: number, encoding?: BufferEncoding);
     do(): Promise<void>;
 }
 
+// @public
+abstract class Credential_2 implements RequestPolicyFactory {
```

Comment:
should this be named StorageCredential instead?

Reply:
Credential is a [web type](https://developer.mozilla.org/en-US/docs/Web/API/Credential) so the api extractor is trying to de-dup here. But generally speaking, the feedback here is to prefix the type with `Storage` since it is specific to storage and is not an SDK-wide type.

Reply:
Was this type exported to the end users? If so, then disregard my comment.

Reply:
Yes, it is exported and is the base of all other concrete credentials.  It could be an api-extractor bug where they don't show `export` for abstract classes in api.md.